types/expo/expo-tests.tsx

```diff
@@ -569,6 +569,10 @@ async () => {
     </Svg>
 );
 
+() => (
```

Comment:
Why not adding it to the [previous test](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/52b7f7abc046886678d001cfa6f499cc7c35d416/types/expo/expo-tests.tsx#L464)?

Reply:
Because "preserveAspectRatio" is an optional attribute on the "Svg" root element and I wanted the test to cover both the "attribute present" and "attribute not present" cases.